agendas/2023/02-Feb/02-wg-primary.md

```diff
@@ -135,3 +136,8 @@ hold additional secondary meetings later in the month.
 1. `@defer`: fully deduplicate response (30m, Ivan)
    - [RFC] (https://github.com/robrichard/defer-stream-wg/discussions/65#discussioncomment-4757697)
    - GraphQL.js implementation: TBD
+1. Default Value Validation Status Check (5m, Yaacov)
+   - [Lee's original PR stack on graphql-js](https://github.com/graphql/graphql-js/pull/3049)
+   - [Now rebased on main](https://github.com/graphql/graphql-js/pull/3814)
+   - [Benjie's original PR for graphql-spec](https://github.com/graphql/graphql-js/pull/3049)
+   - [Now rebased on main](https://github.com/graphql/graphql-spec/pull/793)
```

Comment:
These links appear to be pointing to the wrong places.

I've also just merged `main` into my branch so your rebase isn't needed - best to keep the discussion in one place.

Reply:
Done!

And I closed https://github.com/graphql/graphql-spec/pull/1012